spec/update-process-env-spec.js

```diff
@@ -84,5 +84,33 @@ describe('updateProcessEnv(launchEnv)', function () {
         expect(process.env).toEqual({FOO: 'bar'})
       })
     })
+
+    describe('shells on osx', function () {
+      it('envShouldBePatched() returns the shell when the shell should be patched', function () {
```

Comment:
Could you move the function name up into the `describe`, so that the `it` sentences read a bit better?

```js
describe('envShouldBePatched()', function () {
  it('returns true for shells that ', function () {
    // ...
  })
```